frontend/src/app/styles/app.css

```diff
@@ -0,0 +1,5 @@
+.course-input-container {
+  max-width: 1000px;
+  margin: 40px auto; }
+
+/*# sourceMappingURL=app.css.map */
```

Comment:
we would like to use global variables to keep the style in check. Please use SCSS an define some variables for spacing and margins. Usually they are multiples of 2. Also please use rems instead of pixels, and for large values please use percentages

Reply:
Yes, my bad. Actually the styles are taken only from the scss at this moment.  I will delete that file as soon as I'll fix the other things at the next commit.